attributes/default.rb

```diff
@@ -1,5 +1,6 @@
 # This cookbook support only runit
 default['bosun']['init_style']      = 'runit'
+default['bosun']['install_style']   = 'binary' # binary or golang
```

Comment:
I think installation of Golang and compilation from source should be made more explicit than that.